snap/hooks/pre-refresh

```diff
@@ -3,4 +3,4 @@
 # save this revision for when we run again in the post-refresh
 snapctl set lastrev="$SNAP_REVISION"
 
-snapctl set release="geneva"
+snapctl set release="ireland"
```

Comment:
add newline at end of file

Reply:
We are leaving the snap changes to the Canonical team. They are planning  a re-vamp of it.